client/src/components/NewsWidget/NewsWidget.test.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import ShallowRenderer from "react-test-renderer/shallow";
+import { MenuItem, Select } from "@material-ui/core";
+import NewsWidget from "./";
+import { shallow } from "enzyme";
+
+let component;
+
+beforeEach(() => {
+    component = shallow(<NewsWidget />);
+});
+
+test("it renders correctly", () => {
+    const shallowRenderer = new ShallowRenderer();
+    const snapshotComponent = shallowRenderer.render(<NewsWidget />);
+    expect(snapshotComponent).toMatchSnapshot();
+});
+
+test("it contains Material UI Select, MenuItem components", () => {
+    expect(component.find(Select)).toHaveLength(2);
+    expect(component.find(MenuItem)).toHaveLength(10);
+});
+
+test("dropdown default values load in", () => {
+    expect(component.find(Select).at(0).props().defaultValue).toEqual("general");
+    expect(component.find(Select).at(1).props().defaultValue).toEqual("nz");
+});
```

Comment:
There should be something to test that the component properly tries to get newsarticles from the API, and ideally something to check that the right data gets returned. This way we can know if the endpoint becomes inactive